provider/cmd/pulumi-gen-eks/main.go

```diff
@@ -1460,7 +1565,7 @@ func generateSchema() schema.PackageSpec {
 			}),
 			"java": rawMessage(map[string]interface{}{
 				"dependencies": map[string]string{
-					"com.pulumi:aws":        "6.5.0",
+					"com.pulumi:aws":        "6.18.2",
```

Comment:
https://github.com/pulumi/pulumi-eks/issues/1175 perhaps we could keep it updated more frequently. If we're getting better with keeping AWS backwards-compatible it's probably a good thing to do.